packages/documentation/copy/en/reference/Target Declaration.md

```diff
@@ -41,7 +41,7 @@ A target specification may have optional parameters, the names and values of whi
 - [**runtime-version**](#runtime-version): Specify which version of the runtime system to use.
 - [**rust-include**](#rust-include): (Rust only) A set of Rust modules in the generated project.
 - [**single-file-project**](#single-file-project): (Rust only) If true, enables [single-file project layout](#single-file-layout).
-- [**threading**](#threading): Whether to use multiple threads.
+- [**single-threaded**](#single-threaded): Specify to not use multithreading.
```

Comment:
This change was not merged yet, I believe, because it got stuck in review, so we'll have to move these changes to another branch...

Reply:
I fixed this by moving the changes to the `single-threaded` branch.